src/main/java/edu/uc/inventorymanager/dto/Item.java

```diff
@@ -6,10 +6,10 @@
 
 @RequiredArgsConstructor
 public @Data class Item {
-    @NonNull private int id;
-    private String name;
-    @NonNull private String description;
-    private String location;
+    @NonNull private int itemId;
+    private String itemName;
+    @NonNull private String itemDescription;
+    private String itemLocation;
```

Comment:
“Variables in Java capture the state of the object created from a class. The name of the variable should describe the intent of the variable clearly.” I refactored (renamed) these variables to appropriately match their intention. Id, Name were used repeatedly in several cases when declaring objects.  

Source: https://www.baeldung.com/java-clean-code